src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -29,39 +29,50 @@ const standardSectionFields: GraphQLFieldConfigMap<any, ResolverContext> = {
   },
 }
 
+export const HomeViewSectionTypeNames = {
+  GenericHomeViewSection: "GenericHomeViewSection",
+  ArtworksHomeViewSection: "ArtworksHomeViewSection",
+  ArtistsHomeViewSection: "ArtistsHomeViewSection",
+  HeroUnitsHomeViewSection: "HeroUnitsHomeViewSection",
+  FairsHomeViewSection: "FairsHomeViewSection",
+  ArticlesHomeViewSection: "ArticlesHomeViewSection",
+  MarketingCollectionsHomeViewSection: "MarketingCollectionsHomeViewSection",
+  ShowsHomeViewSection: "ShowsHomeViewSection",
+  ViewingRoomsHomeViewSection: "ViewingRoomsHomeViewSection",
+  ActivityHomeViewSection: "ActivityHomeViewSection",
+  AuctionResultsHomeViewSection: "AuctionResultsHomeViewSection",
+  SalesHomeViewSection: "SalesHomeViewSection",
```

Comment:
```tsx
export const HomeViewSectionTypeNames = {
  HomeViewActivitySection: "HomeViewActivitySection",
  HomeViewArticlesSection: "HomeViewArticlesSection",
  HomeViewArtistsSection: "HomeViewArtistsSection",
  HomeViewArtworksSection: "HomeViewArtworksSection",
  HomeViewAuctionResultsSection: "HomeViewAuctionResultsSection",
  HomeViewFairsSection: "HomeViewFairsSection",
  HomeViewGenericSection: "HomeViewGenericSection",
  HomeViewHeroUnitsSection: "HomeViewHeroUnitsSection",
  HomeViewMarketingCollectionsSection: "HomeViewMarketingCollectionsSection",
  HomeViewSalesSection: "HomeViewSalesSection",
  HomeViewShowsSection: "HomeViewShowsSection",
  HomeViewViewingRoomsSection: "HomeViewViewingRoomsSection",
}
```

(Alphabetized as well) 

Also to the point -- With the introduction of application regions (ie, "HomeSections") we're introducing the potential for _other_ application regions to be represented as types/code/file-names. If we don't prefix the region (the common category), finding things will be much more difficult at a glance.